corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -466,8 +466,23 @@ def _migrate_case_actions(couch_case, sql_case):
 
 
 def _migrate_case_attachments(couch_case, sql_case):
-    # TODO: maybe wait until case attachments are in blobdb
-    pass
+    """Copy over attachment meta """
+    attachments = []
+    for name, attachment in six.iteritems(couch_case.case_attachments):
+        attachments.append(CaseAttachmentSQL(
+            name=name,
+            case=sql_case,
+            identifier=attachment.identifier,
+            attachment_src=attachment.attachment_src,
+            attachment_from=attachment.attachment_from,
+            content_type=attachment.server_mime,
+            content_length=attachment.content_length,
+            blob_id=attachment.id,
+            blob_bucket=couch_case._blobdb_bucket(),
+            properties=attachment.attachment_properties,
+            md5=attachment.server_md5
+        ))
+    sql_case.unsaved_attachments = attachments
```

Comment:
Strangely this will differ from forms. you'll need to do this for each attachment: `sql_case.track_create(attachment)`

Reply:
Cool, I'm seeing the track_create flow now. Do you know if there a reason for the separate code paths or is it just that different people created each?

c368a5c

Reply:
not 100% sure, I think there may have been a reason at some point but a quick look through the code didn't reveal anything obvious.